spiketoolkit/postprocessing/postprocessing_tools.py

```diff
@@ -39,21 +40,25 @@ def get_unit_waveforms(recording, sorting, unit_ids=None, grouping_property=None
     save_as_features: bool
         If True (default), waveforms are saved as features of the sorting extractor object
     compute_property_from_recording: bool
-        If True and 'grouping_property' is given, the property of each unit is assigned as the corresponding propery of
-        the recording extractor channel on which the average waveform is the largest
+        If True and 'grouping_property' is given, the property of each unit is assigned as the corresponding property
+        of the recording extractor channel on which the average waveform is the largest
     verbose: bool
         If True output is verbose
     seed: int
         Random seed for extracting random waveforms
-    return_idxs: list
-        List of indexes of used spikes for each unit
+    memmap: bool
```

Comment:
This kargs is ambiguous. It could refer to the file itself.
Maybe use_memmap or something like that ?

Reply:
ok

Reply:
mmm I actually like memmap. As long as it's in the docs!It's like: memmap?